runtime/shared_heap.c

```diff
@@ -199,7 +199,27 @@ void caml_teardown_shared_heap(struct caml_heap_state* heap) {
               released, released_large);
 }
 
-/* this _must_ be called either with the pool_freelist.lock held or
+/* TODO: resurrect major_heap_increment */
+
+uintnat new_chunk_bsize(void)
```

Comment:
`alloc_pool` should now call this function rather than duplicating the logic?

(Also, it looks like the `caml_pool_min_chunk_bsz` logic from 3384 got lost in a merge, possibly of compaction. This restores it, which is another good reason to fix `alloc_pool`)

Reply:
Yes, that's an oversight. Fixed.